src/main/data/studios.json

```
@@ -1318,6 +1318,16 @@
     "description": "Golden Touch Film",
     "url": "https://www.goldentouchfilm.com"
   },
+  {
+    "code": "GYF",
+    "contact": {
+      "address": "Building 2, No.6 Chaoyang Park South Road, Chaoyang District, Beijing, China",
```

```suggestion
      "address": "Building 2, No. 6 Chaoyang Park South Road, Chaoyang District, Beijing, China",
```

There was no postal code?